site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Storage and Cookies"
+seoTitle: "Chrome Extensions storage and cookies"
+date: 2023-08-01
+description: Overview of how web storage APIs and cookies work in extensions
+---
+
+Extensions can store cookies and access web storage APIs similar to a normal website. However, in
+some cases these behave differently in extensions.
+
+This article does not cover extension specific APIs. For more on these, see
+[chrome.storage][chrome-storage-api].
+
+## Storage {: #storage }
+
+### Persistence {: #storage-persistence }
+
+Extension storage is always persistent. For example, when a user uses the "Clear Browsing Data"
+feature, this does not affect extensions. This includes any data stored using web storage APIs (such
+as Local Storage and IndexedDB). Additionally, extensions are not subject to storage eviction under
+heavy memory pressure.
+
+By default, extensions are subject to the normal quota restrictions on storage, which can be checked
+using the `navigator.storage.estimate()` method. However, they can request the `unlimitedStorage`
+permission which affects both extension and web storage APIs.
+
+### Access in service workers {: #storage-in-service-workers }
```

Comment:
Don't extensions behave in this case exactly like web pages?  Do we need to say much in this case? The offscreen workaround feels a bit misplaced here as the intention of the article is to explain how storage works.

Reply:
I still think it'd be better to move this content to the migration guide and link from here instead.